code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -191,12 +193,30 @@ func (c *ChallengeEntity) getCommitTransaction() (*transaction.Transaction, erro
 	tx := datastore.GetStore().GetTransaction(ctx)
 
 	createdTime := common.ToTime(c.CreatedAt)
+
+	logging.Logger.Info("[challenge]verify: ",
+		zap.Any("challenge_id", c.ChallengeID),
+		zap.Time("created", createdTime))
+
+	network := zcncore.GetNetwork()
```

Comment:
move this call into the gosdk

Reply:
We need to getNetwork here only. I tried moving in gosdk and we are getting nil.